docs/contributing/documentation-guidelines/index.md

```diff
@@ -1,5 +1,8 @@
 # Documentation guidelines
 
-!!! warning
+## Instructions to add documentation
 
-    Under Construction
+There are two ways to contribute to the documentation page.
```

Comment:
This page isn't for how to contribute, but how to write good documents.
So you should edit https://autowarefoundation.github.io/autoware-documentation/main/contributing/#contribution-workflows or https://autowarefoundation.github.io/autoware-documentation/main/help/support-guidelines/#github-discussions instead.

Reply:
That's what I wanted to know